### PR TITLE
Add infomation about export grids to CSV

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: concepts/05 UI Components/DataGrid/00 Getting Started with DataGrid/80 Export Data to Excel.md [hunk 1]
```diff
@@ -207,6 +207,8 @@ This method can be called at any point in your application. The following code c
 
 Run the code and click the Export button in the upper-right corner. Grid data should be exported to the DataGrid.xlsx file.
 
+[note]You can also export DataGrid to CSV. To do this, implement the **exportDataGrid(options)** method in the same way as shown in the following ticket: <a href="https://supportcenter.devexpress.com/ticket/details/t920593/pivotgrid-exceljs-export-to-export-pivotgrid-into-csv-file" target="_blank">Export PivotGrid into CSV file</a>.
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/DataGrid/00 Getting Started with DataGrid/80 Export Data to Excel.md]: I see this note is repeated in three places. How about an include?

Comment (AlexJameson) [reply concepts/05 UI Components/DataGrid/00 Getting Started with DataGrid/80 Export Data to Excel.md]: @RomanTsukanov I realized that I cannot test the includes render at the moment on this branch. can we put an include inside a note?

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Components/dxDataGrid/1 Configuration/export/export.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ type: Object
 ---
 ---
 ##### shortDescription
+[tags] xlsx, csv
+
 Configures client-side exporting.
 
 ---
```
Comment (albertov05) [inline api-reference/10 UI Components/dxDataGrid/1 Configuration/export/export.md:11]: Can we update line 10 (below) as follows?

`A user can click the **Export** button to save an Excel file with the exported data. Data types, sort, filter, and group settings are maintained.`

File: api-reference/10 UI Components/dxPivotGrid/1 Configuration/export/export.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ type: Object
 ---
 ---
 ##### shortDescription
+[tags] xlsx, csv
+
 Configures client-side exporting.
 
 ---
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPivotGrid/1 Configuration/export/export.md:11]: Can we update line 10 (below) as follows?

`A user can click the **Export** button to save an Excel file with the exported data. Data types, sort, filter, and group settings are maintained.`

**Pull request merged**